File: projet/src/main/java/com/enicar/projet/dtos/PlanificationGroupeDTO.java
package com.enicar.projet.dtos;

import lombok.Data;

import java.time.LocalDate;
import java.time.LocalTime;
import java.util.List;

@Data
public class PlanificationGroupeDTO {

    private String groupe;
    private String niveau;

    private LocalDate date;
    private LocalTime heureDebut;
    private int duree;

    private Long salleId;
    private String statut;

    private List<MembreJuryDTO> membresJury;
    private List<Long> etudiantIds;
}

File: front_spring/src/app/models/PlanificationGroupeDTO.ts
import { MembreJury } from "./membre-jury";

export interface PlanificationGroupeDTO {
  groupe: string;
  niveau: string;
  date: string;
  heureDebut: string;
  duree: number;
  salleId: number;
  statut: string;
  membresJury: MembreJury[];
  etudiantIds: number[];
}
 